django/core/checks/model_checks.py

```diff
@@ -128,6 +97,29 @@ def field_error(model_key, func, args, keywords):
         }
         return Error(error_msg % params, obj=keywords['field'], hint=None, id='fields.E020')
 
+    def signal_connect_error(model_key, func, args, keywords):
+        error_msg = (
+            "%(receiver)s was connected to the '%(signal)s' signal with a "
+            "lazy reference to the sender '%(model)s', but %(model_error)s."
```

Comment:
looks like the wording of this message should be updated in `docs/ref/checks.txt`